torch/_inductor/simple_fsdp/bucket.py

```diff
@@ -4,7 +4,7 @@
 import torch
 import torch.distributed as dist
 
-from .. import ir, scheduler
+from .. import ir, scheduler, config
```

Comment:
let's not import all of `config` if you only need `config.simplefsdp`
can we do something like `from ..config.simplefsdp import fsdp_degree`

Reply:
I was having problem importing `..config.simple fsdp`, I changed it to `from ..config import simplefsdp`, and call `simplefsdp.fsdp_degree`
`